.travis.yml

```diff
@@ -24,27 +23,18 @@ env:
     - JOBS=1
 
 before_install:
-  - npm install -g npm@5
   - npm config set spin false
 
 script:
   - npm run lint:js
   - npm test
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then ember deploy production --activate; fi'
```

Comment:
Lets add this step back and convert it to `ember build --prod`

Reply:
but that doesn't do anything 😄 the actual deployment is done via this command https://github.com/ember-learn/guides-app/pull/52/files#diff-354f30a63fb0907d4ad57269548329e3R40 and when we move to using Heroku the build on Travis has nothing to do with the deployment process.

It might be best for us to hop on a call if you would like me to explain the build process to you 👍